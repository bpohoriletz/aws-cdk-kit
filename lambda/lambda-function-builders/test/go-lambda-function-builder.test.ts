import GoLambdaFunctionBuilder from '../go-lambda-function-builder';

describe('new GoLambdaFunctionBuilder', () => {
  it('has correct defaults', () => {
    expect(new GoLambdaFunctionBuilder()).toMatchSnapshot();
  });
});
