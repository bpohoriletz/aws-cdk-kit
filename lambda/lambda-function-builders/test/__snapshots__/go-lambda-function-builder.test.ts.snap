// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new GoLambdaFunctionBuilder has correct defaults 1`] = `
GoLambdaFunctionBuilder {
  "props": LambdaFunctionProduct {},
}
`;
